.git-pre-commit

```diff
@@ -10,7 +10,8 @@ import sys
 
 
 def main():
-    out = subprocess.check_output("git diff --cached --name-only", shell=True)
+    out = subprocess.check_output("git diff --cached --name-only", shell=True,
+                                                      universal_newlines=True)
```

Comment:
Converting the string arguments to bytes would work correct the error too.

`files = [x for x in out.split(b'\n') if x.endswith(b'.py') and`